roles/ovirt.hosts/filter_plugins/filter.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/python
+class FilterModule(object):
+    def filters(self):
+        'Define filters'
+        return {
+            'removepassword': self.removepassword,
```

Comment:
Can you please rename it to `removesensitive` or `withoutsensitive`, let make it generic, so it's reusable.